PowerEditor/installer/signCert/npp_sign_cert.pfx

```diff
@@ -0,0 +1 @@
+Replace with your actual cert or test cert
```

Comment:
Remove both `npp_sign_cert.pfx` and `SignTool.exe` files.
They are not necessary, and should not be in the repository.